services/brig/test/integration/API/User/Util.hs

```diff
@@ -551,3 +551,25 @@ setTeamFeatureLockStatus galley tid status =
 
 lookupCode :: MonadIO m => DB.ClientState -> Code.Key -> Code.Scope -> m (Maybe Code.Code)
 lookupCode db k = liftIO . DB.runClient db . Code.lookup k
+
+getNonce ::
+  (MonadIO m, MonadHttp m) =>
+  Brig ->
+  UserId ->
+  m ResponseLBS
+getNonce = nonce get
+
+headNonce ::
+  (MonadIO m, MonadHttp m) =>
+  Brig ->
+  UserId ->
+  m ResponseLBS
+headNonce = nonce Bilge.head
+
+nonce :: ((Request -> c) -> t) -> (Request -> c) -> UserId -> t
```

Comment:
This is fine, just FYI I think there is also a combinator `method` that lets you change the method in the function sequence that also contains `paths`, `zUser`, ...